Content.Server/Atmos/EntitySystems/AirtightSystem.cs

```diff
@@ -104,6 +106,9 @@ public void InvalidatePosition(GridId gridId, Vector2i pos, bool fixVacuum = fal
             if (!gridId.IsValid())
                 return;
 
+            var query = EntityManager.GetEntityQuery<AirtightComponent>();
+            _explosionSystem.UpdateAirtightMap(gridId, pos, query);
```

Comment:
part of me feels like the airtight map can be used for a lot of other stuff and it should just be part of `AirtightSystem`, but im not sure

Reply:
The explosion map does have some explosion-specific info (how much damage it takes to break the entity), but it could definitely be useful things like smoke, puddles, or kudzu.